lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || [ 14, 15, 16, 17, 18, 19 ];
```

Comment:
right,  it's overridable though.
Should this be a constant at the top to point it out?